### PR TITLE
Add Razor SKALE Chain

### PR DESCRIPTION
**Pull request by rajkharvar**
Changes proposed in this pull request:
- Add support for Razor SKALE Chain

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -1523,5 +1523,19 @@
...
+    "name": "Razor SKALE Chain",
+    "shortName": "RazorSchain",
+    "chainId": 278611351,
+    "network": "mainnet",
+    "multicall": "0x295762C6f27c33cD7ce58fB6667E841E4EA67ef1",
+    "rpc": [
+      "https://mainnet.skalenodes.com/v1/turbulent-unique-scheat"
+    ],
+    "explorer": "https://turbulent-unique-scheat.explorer.mainnet.skalenodes.com/",
+    "start": 327242,
```
Comment (ChaituVR) [inline src/networks.json:1538]:
```suggestion
    "start": 401478,
```

File: src/networks.json [hunk 2]
```diff
@@ -1523,5 +1523,19 @@
...
+    "key": "278611351",
+    "name": "Razor SKALE Chain",
+    "shortName": "RazorSchain",
+    "chainId": 278611351,
+    "network": "mainnet",
+    "multicall": "0x295762C6f27c33cD7ce58fB6667E841E4EA67ef1",
+    "rpc": [
+      "https://mainnet.skalenodes.com/v1/turbulent-unique-scheat"
+    ],
+    "explorer": "https://turbulent-unique-scheat.explorer.mainnet.skalenodes.com/",
```
Comment (ChaituVR) [inline src/networks.json:1537]: Can't use https://razorscan.io/ ?

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ChaituVR** [comment]
Everything else looks good :)

File: src/networks.json (hunk 1 above)
Comment (rajkharvar) [reply src/networks.json]: Hey @ChaituVR
Just to confirm isn't the `start` is Multicall contract creation block?

File: src/networks.json (hunk 2 above)
Comment (rajkharvar) [reply src/networks.json:1537]: https://razorscan.io is for validators and delegators to participate in the network whereas https://turbulent-unique-scheat.explorer.mainnet.skalenodes.com/ is the actual chain explorer.

File: src/networks.json (hunk 1 above)
Comment (ChaituVR) [reply src/networks.json]: Yep, correct :)

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**